src/sentry/quotas/redis.py

```diff
@@ -95,16 +101,20 @@ def get_next_period_start(interval):
         client = self.cluster.get_local_client_for_key(six.text_type(project.organization.pk))
         rejections = is_rate_limited(client, keys, args)
         if any(rejections):
+            enforce = False
             worst_case = (0, None)
             for quota, rejected in zip(quotas, rejections):
                 if not rejected:
                     continue
+                if quota.enforce:
+                    enforce = True
+                    continue
```

Comment:
I don't think this `continue` statement is necessary, since it prevents the `worst_case` values from being set when the quota is actually enforced.